.circleci/config.yml

```diff
@@ -1,6 +1,46 @@
 version: 2
+workflows:
+  version: 2
+  everything:
+    jobs:
+      - unit_test:
+          filters:
+            branches:
+              ignore:
+                - gh-pages
+      - openapi_validation
 jobs:
-  build: # runs not using Workflows must have a `build` job as entry point
+  openapi_validation:
+    docker: # run the steps with Docker
+      - image: circleci/openjdk:11.0.4-jdk-stretch
+        environment:
+          JAVA_TOOL_OPTIONS: -Xmx512m # Java can read cgroup. Sadly the cgroup in CircleCI is wrong. Have to manually set. Nothing to do with surefire plugin, it has its own JVM. The two of these must add up to a bit less than 4GB.
```

Comment:
Is this a place to use a resource_class instead -- are we trying to make sure things fit into 4GB? Or does it work fine as is?

Reply:
It's copied from the other job